pre-commit.sh

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env bash
 
-if [ "$1" == "--no-browser" ]
+if [ "$1" == "--no-parallel" ]
 then
-    SHOW_BROWSER=false
+    SHOW_BROWSER=true
+    INSTANCES=1
 fi
 
 set -eu
 
 bundle --quiet
 mkdir -p testreport
-SHOW_BROWSER=${SHOW_BROWSER:-"true"} TEST_ENV=local bundle exec cucumber --strict
\ No newline at end of file
+SHOW_BROWSER=${SHOW_BROWSER:-"false"} TEST_ENV=local bundle exec parallel_cucumber features/ -n ${INSTANCES:-3} -o "--strict"
```

Comment:
Don't need to quote `false` or `--strict`

Reply:
I think it does. The strict is being passed down to the `cucumber` via `parallel_cucumber`s parameter `-o`

Reply:
bash will get to the `"`s before cucumber ever sees them

Reply:
If you wanted to pass a space-separated string as a single parameter then yes, but you're not, so you don't need to

Reply:
oh true, still it makes it more readable that `--strict` is a parameter for `-o` and not for `parallel_cucumber`. I'll fix the `false`